s/features/notes/components/notes/xiome-notes.ts

```diff
@@ -102,14 +117,30 @@ export class XiomeNotes extends ComponentWithShare<{
 		`
 	}
 
-	render() {
-		return renderOp(this.#model.state.accessOp, access =>
-			access?.user
+	#renderBasedOnStateOfNotesArray (){
+		const {old, notesOp} = this.#cache.cacheState
+		return html`
+			${ops.value(notesOp)?.length === 0
 				? html`
+					${this.#renderTabs()}
+					<slot name="empty notes">
```

Comment:
- let's avoid having any spaces in any slot name. let's just call this slot `empty`